transport/p2p/src/constants.rs

```diff
@@ -1,5 +1,5 @@
 /// Connection idle timeout for all protocols used in the swarm.
-pub const HOPR_SWARM_IDLE_CONNECTION_TIMEOUT: std::time::Duration = std::time::Duration::from_secs(300); // 5 minutes
+pub const HOPR_SWARM_IDLE_CONNECTION_TIMEOUT: std::time::Duration = std::time::Duration::from_secs(10);
```

Comment:
Why?

Reply:
Closer to the libp2p default, also what other projects use: https://github.com/libp2p/rust-libp2p/blob/ad9a1b27587f384f644f30f8fccb2b9e6ad47edb/swarm/src/connection/pool.rs#L1000